backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java

```diff
@@ -73,10 +91,27 @@ public PerspectiveManager(PerspectiveAPI perspectiveAPI) {
    * @return An object describing the recommendation resulting from the analysis.
    * @throws NoAppropriateStoryException if story is not considered appropriate
    */
-  public StoryDecision generateDecision(String story) {
-    // TODO: Replace this fake behavior with a real API call, etc.
+  public StoryDecision generateDecision(String story) throws NoAppropriateStoryException {
+    PerspectiveAPIClient apiClient = new PerspectiveAPIClient(perspectiveAPI);
+    PerspectiveValues storyValues = apiClient.analyze(REQUESTED_ATTRIBUTES, story);
+    boolean decision = ContentDecisions.makeDecision(storyValues);
```

Comment:
IMO `boolean isStoryAppropriate` would be clearer than `boolean decision`.

In conversational language, "Decision? True" is a bit confusing. "Is story appropriate? True" is clearer.

Reply:
That makes sense. I'll change this.

Reply:
Done